src/sockets/base.js

```diff
@@ -26,6 +26,8 @@ export default (io) => {
       .sort('-createdAt')
       .then();
 
+    socket.on('message', () => io.emit('message'));
+
     socket.emit('connection', { latestMessage });
```

Comment:
No, it actually does work without it. My bad, noticed just now, that the same emit is already here https://github.com/Sirimangalo/meditation-plus-server/blob/master/src/app/routes/_message.router.js#L113 . We should revert this merge